README.md

```diff
@@ -249,9 +249,15 @@ First of all, you need a compiler with decent C11 support. The following compile
 - MSVC >= 2015
 - XCode Clang >= 6.0
 
-Pony requires LLVM version 3.9.1.
+When building from sources by default Pony uses LLVM as installed in you system or you may also build LLVM from sources. In either case the supported version of LLVM is 3.9.1 with experimental support for version 6.0.0. Versions other than LLVM 3.9.1 may result in decreased performance or crashes in generated applications.
```

Comment:
as the long-term plan for ponyc is to vendor LLVM, should we maybe make compiling the sources the default?
Or do we keep this PR as it is, basically adding it as an option and switch defaults later?

I would also prefer to have 1 Makefile for everything, maybe using a variable to switch between some llvm on the system and the sources. But I fully understand that this might be a major effort that might actually not be worth it.